website/versions.json

```diff
@@ -1,4 +1,5 @@
 [
+  "2.5.0",
```

Comment:
Not sure why but you are still missing these changes: https://github.com/openfun/richie/pull/1343/files#diff-5348aab53dc03b67a95ede93b246aa31f9e2a642e74b95c314025e0c4f8e0489R201

Reply:
This is dynamic content, nothing is generated from it. But if I start website locally, I see these changes.